### PR TITLE
Rename batchChange module/controller/service to dnsChange

### PR DESCRIPTION
**Pull request by JoshKessler**
Fixes https://github.com/vinyldns/vinyldns/issues/873

Changes in this pull request:
- Renames module, controllers, service and directive in modules/portal/public/lib/dns-change/ from BatchChange to DnsChange
- Updates their uses and references.

### CODE REVIEW

**Comment by britneywright** [comment]
Thanks for the contribution @JoshKessler!

File: modules/portal/public/lib/dns-change/dns-change-new.controller.js [hunk 1]
```diff
@@ -100,7 +100,7 @@
 
                 formatData(payload);
 
-                return batchChangeService.createBatchChange(payload, true)
```
Comment (mitruly) [inline modules/portal/public/lib/dns-change/dns-change-new.controller.js]: The `handleError` message should also switch `batchChangesService` to `dnsChangesService`.

**Comment by JoshKessler** [comment]
@mitruly, I've updated the missed reference. I thought I'd both pushed the change with my non-Comcast GH account and signed the CLA with that one. When I tried to access the CLA just now and made sure to sign in with my personal account, it says "You have signed the CLA for vinyldns/vinyldns." If it stil looks wrong on your end, I may need some guidance with that.

**Comment by mitruly** [comment]
@JoshKessler I believe this is due to your e-mail address in your `~/.gitconfig` not matching your open source Github account. Can you try the following steps:
1. Correct your `email` in `~/.gitconfig`
2. Rebase with master and push your branch so the new settings will get recognized.

**Pull request merged**